tests/unit/utils/containers.go

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import (
+	cassdcV1beta1 "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
```

Comment:
minor nit - can you make the aliasas all lower case. That seems to be the common convention.